pretty_gpx/city/wip_city_sample.py

```diff
@@ -67,12 +68,24 @@ def plot(gpx_track: GpxTrack, theme_colors: ThemeColors) -> None:
     title = TextData(x=b.lon_center, y=b.lat_max - 0.8 * b.dlat * layout.title_relative_h,
                      s="Marathon de Paris", fontsize=mm_to_point(20.0),
                      fontproperties=FontProperties(fname=os.path.join(FONTS_DIR, "Lobster 1.4.otf")),
-                     ha="center")
-    stats_text = f"{gpx_track.list_cumul_dist_km[-1]:.2f} km - {int(gpx_track.uphill_m)} m D+"
-    stats = TextData(x=b.lon_center, y=b.lat_min + 0.5 * b.dlat * layout.stats_relative_h,
-                     s=stats_text, fontsize=mm_to_point(18.5),
-                     fontproperties=FontProperties(fname=os.path.join(FONTS_DIR, "Lobster 1.4.otf")),
-                     ha="center")
+                     ha="center",
+                     va="center")
+    
+    if gpx_track.duration_s is not None:
+        stats_text = f"{gpx_track.list_cumul_dist_km[-1]:.2f} km - {int(gpx_track.uphill_m)} m D+"
+        stats_text += f"\n{format_timedelta(gpx_track.duration_s)}"
+        stats = TextData(x=b.lon_center, y=b.lat_min + 0.5 * b.dlat * layout.stats_relative_h,
+                        s=stats_text, fontsize=mm_to_point(18.5),
+                        fontproperties=FontProperties(fname=os.path.join(FONTS_DIR, "Lobster 1.4.otf")),
+                        ha="center",
+                        va="center")
+    else:
+        stats_text = f"{gpx_track.list_cumul_dist_km[-1]:.2f} km - {int(gpx_track.uphill_m)} m D+"
+        stats = TextData(x=b.lon_center, y=b.lat_min + 0.5 * b.dlat * layout.stats_relative_h,
```

Comment:
use the conditional to create the `stats_text` but then write a single `TextData` instanciation